test/SmokeTests/OOProcSmokeTests/DotNetIsolated/HelloCitiesUntyped.cs

```diff
@@ -38,18 +39,19 @@ internal static class HelloSequenceUntyped
     /// <summary>
     /// Orchestrator function that calls the <see cref="SayHelloUntyped"/> activity function several times consecutively.
     /// </summary>
-    /// <param name="state">The opaque orchestration state that gets passed to the function. Code should never attempt to read this state.</param>
+    /// <param name="requestState">The serialized orchestration state that gets passed to the function.</param>
     /// <returns>Returns an opaque output string with instructions about what actions to persist into the orchestration history.</returns>
     [Function(nameof(HelloCitiesUntyped))]
-    public static string HelloCitiesUntyped([OrchestrationTrigger] string state) =>
-        DurableOrchestrator.LoadAndRun<string, string>(state, async (context, _) =>
+    public static string HelloCitiesUntyped([OrchestrationTrigger] string requestState, FunctionContext functionContext) =>
+        OrchestrationRunner.LoadAndRun<string, string>(requestState, async (context, _) =>
         {
             string result = "";
             result += await context.CallActivityAsync<string>(nameof(SayHelloUntyped), "Tokyo") + " ";
             result += await context.CallActivityAsync<string>(nameof(SayHelloUntyped), "London") + " ";
             result += await context.CallActivityAsync<string>(nameof(SayHelloUntyped), "Seattle");
             return result;
-        });
+        },
+        functionContext.InstanceServices);
```

Comment:
Passing `InstanceServices` allows us to get access to injected dependency services, which we didn't have before. In the final version of .NET Isolated support, developers hopefully won't have to worry about this or `OrchestrationRunner` at all.

Reply:
Can you say a bit more about this? What services are we accessing in this case? This doesn't mean that customers are now allowed to use DI in their orchestrations, right?

Reply:
Correct, customers are still not allowed to use DI in orchestrations. The primary use-cases are getting the Azure Function's `ILoggerFactory` for logging and for supporting custom `IDataConverter` implementations. Without this, customers wouldn't be able to use `ILogger` in their orchestrator code, nor could they override our serialization logic for inputs and outputs.